pkg/clients/unstructured_test.go

```diff
@@ -31,6 +31,7 @@ type podBasic struct {
 }
 
 // PodBasic returns a corev1.Pod struct and a corresponding Unstructured struct.
+//nolint: golint
```

Comment:
LGTM. any reason we're not linting here and the 2 other `nolint` spots?

Reply:
It didn't like that we're returning a reference to an unexported struct, but it's working for now, and I didn't want to change it.